Pipfile

```diff
@@ -13,6 +13,5 @@ python-keycloak-client = "==0.2.3"
 
 [dev-packages]
 pylint = "==2.6.2"
-
-[requires]
-python_version = "3.8"
```

Comment:
Just curiosity, why don't we specify both `3.8` and `3.9` instead of removing the rule?
Or something like `>3.8` if that exists of course.

Reply:
I wish that was possible, but pipenv does not support specifying more than one version of python. You either have to specify one version or none at all. There are open issues about this in pipenv, but the maintainers have said they have no plans to ever support this.

Reply:
Okey! No problem thanks a lot for the information :)